app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -3352,6 +3355,85 @@ class DatabaseMigrationAndroidTest {
         "deletedAt")
     )
   }
+
+  @Test
+  fun migration_from_48_to_49_create_only_one_encounter_for_per_day_but_add_encounterUuid_to_all_BPs() {
+    //given
+    val tableName = "BloodPressureMeasurement"
+
+    val uuidForFirstBP = "3eb96138-5270-419c-bcdf-86db95d0bc8b"
+    val uuidForSecondBP = "a2455b0c-cf14-46c9-9f73-6e18972982b0"
+    val facilityUuid = "0086ba86-314c-49d9-8822-2563e53bed92"
+    val patientUuid = "c4233170-feb2-3d34-8115-98fef5a68837"
+    val recordedAtForFirstBP = "2018-07-03T00:00:00Z"
+    val recordedAtForSecondBP = "2018-07-03T01:30:00Z"
+    val createdAtForFirstBP = "2018-07-03T00:00:00Z"
+    val createdAtForSecondBP = "2018-07-03T01:30:00Z"
```

Comment:
Can we do `val createdAtForFirstBp = recordedAtForFirstBP`? and `recordedAtForSecondBP` as well?

Reply:
We can do that but we don't expect both these values to be the same always.
But I understand what you mean. I will change the `Instant` for both `createdAt`.